### DISCUSSION TITLE
Cam Pan v2 install?

### DISCUSSION THREAD

**Discussion by inyourowntime** [gtxaspec/wz_mini_hacks#503] [Q&A]
I have tried for days to install on Cam Pan v2

Fat32 format card
added ssh-rsa key
enable web server in config file
get "connection refused"

Am I missing something for the Cam Pan v2?

**Comment by bjs-pdx** [comment]
Connection refused to what?  SSH?  Web? RTSP?  Assume you heard the camera say it was running wz_mini on first boot?  Did you look at #502 for more info on SSH configuration?
